uniffi_bindgen/src/bindings/python/templates/ErrorTemplate.py

```diff
@@ -23,6 +24,21 @@ def __repr__(self):
     class {{ variant_type_name }}(_UniffiTemp{{ type_name }}):
         {%- call py::docstring(variant, 8) %}
 
+    {%-     if variant.has_nameless_fields() %}
+        def __init__(self, *values):
+            if len(values) != {{ variant.fields().len() }}:
+                raise TypeError(f"Expected a tuple of len {{ variant.fields().len() }}, found len {len(values)}")
```

Comment:
Maybe this should say `Expected N arguments` instead, since `values` is packed from the arguments.